typings/balena-sdk.d.ts

```diff
@@ -857,6 +863,7 @@ declare namespace BalenaSdk {
 					uuidOrId: string | number,
 					{ force }: { force?: boolean },
 				): Promise<void>;
+				getSupervisorState(uuidOrId: string | number): Promise<void>;
```

Comment:
I don't think that `void `describes the result of this method.

Reply:
Updated, good catch!